americangut/per_sample.py

```diff
@@ -247,6 +258,57 @@ def taxa_summaries(opts, sample_ids):
     return results
 
 
+def alpha_plot(opts, sample_ids):
+    """Produces digestable alpha diversity distribution plots per sample
+
+    Parameters
+    ----------
+    opts : dict
+        A dict of relevant opts.
+    sample_ids : Iterable of str
+        A list of sample IDs of interest
+
+    Returns
+    -------
+    dict
+        A dict containing each sample ID and any errors observed or None if
+        no error was observed for the sample. {str: str or None}
+    """
```

Comment:
This is the documentation is modeled after every other function in this library.